src/main/kotlin/com/amazon/ionelement/api/IonUtils.kt

```diff
@@ -42,6 +42,8 @@ public fun IonElement.toIonValue(factory: ValueFactory): IonValue {
  * Bridge function that converts from the mutable [IonValue] to an [AnyElement].
  *
  * New code that does not need to integrate with uses of the mutable DOM should not use this.
+ *
+ * This will fail for IonDatagram if the IonDatagram does not contain exactly one user value.
```

Comment:
🗺️ This was a non-obvious edge case that I ran into while testing my changes, so I figured I'd add a note here.